client/src/neko/base.ts

```diff
@@ -202,7 +202,7 @@ export abstract class BaseClient extends EventEmitter<BaseEvents> {
     this._peer = new RTCPeerConnection()
     if (lite !== true) {
       this._peer = new RTCPeerConnection({
-        iceServers: [{ urls: servers }],
+        iceServers: JSON.parse(servers),
```

Comment:
I'd say, having it as data structure passing to client would be nicer. Now it is JSON carried in JSON as string.

Reply:
You are right. Now its cleander